### PR TITLE
Day one hanoi tower

### PR DESCRIPTION
**Pull request by shouya**

### CODE REVIEW

File: 001-hanoi-tower/shou.hs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+hanoi :: Int -> [(Int, Int)]
+hanoi n = hanoi' n 1 3 2
+
```
Comment (crescat) [inline 001-hanoi-tower/shou.hs:3]: two blank lines between functions

Comment (shouya) [reply 001-hanoi-tower/shou.hs:3]: advice taken :)

**Comment by crescat** [comment]
approved : )

**Review by crescat — APPROVED** [review]
Approved!

**Pull request merged**